docs/kubeadm.md

```diff
@@ -0,0 +1,249 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart will show you how to easily install a secure Kubernetes cluster on any machines running Linux, using a tool called `kubeadm` which is part of Kubernetes.
```

Comment:
perhaps go into detail (any requirements for kubeadm?ex. any machine with kernel version blah blah running docker?)